batch-oai-harvest/src/main/java/data/config/OaiHarvestJobConfig.java

```diff
@@ -27,46 +32,71 @@
 @EnableTask
 public class OaiHarvestJobConfig {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
-  public static final String BATCH_JOB = OAI_HARVEST.name();
-  public static final String STEP_NAME = "oaiHarvestStep";
+    private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+    public static final String BATCH_JOB = OAI_HARVEST.name();
+    public static final String IDENTIFIERS_HARVEST_STEP_NAME = "identifiersHarvest";
+    public static final String RECORDS_HARVEST_STEP_NAME = "recordsHarvest";
+
+    @Value("${oaiharvest.chunk.size}")
+    public int chunkSize;
+    @Value("${oaiharvest.parallelization.size}")
+    public int parallelization;
+
+    @Bean
+    public Job oaiHarvestJob(
+            JobRepository jobRepository,
+            @Qualifier("oaiIdentifiersHarvestStep") Step oaiIdentifiersHarvestStep,
+            @Qualifier("oaiRecordsHarvestStep") Step oaiRecordsHarvestStep) {
+
+        LOGGER.info("Chunk size: {}, Parallelization size: {}", chunkSize, parallelization);
+        return new JobBuilder(BATCH_JOB, jobRepository)
+//                .incrementer(new TimestampJobParametersIncrementer())
```

Comment:
Removing this means that we cannot re-run a job with the same parameters. I would re-enable it for now and we can create another ticket to maybe solve this issue differently, as mentioned in ticket MET-5883.

Reply:
You are right. I committed it by mistake. Reverted.